fastly/event_logs.go

```diff
@@ -22,7 +24,7 @@ type Event struct {
 	Metadata    map[string]interface{} `jsonapi:"attr,metadata,omitempty"`
 	ServiceID   string                 `jsonapi:"attr,service_id"`
 	UserID      string                 `jsonapi:"attr,user_id"`
-	CreatedAt   string                 `jsonapi:"attr,created_at"`
+	CreatedAt   *time.Time             `jsonapi:"created_at"`
```

Comment:
> This struct is the only one that uses a different unmarshalling package. The other uses mapstructure and this one uses jsonapi. That is strange, but not the problem.

It actually makes sense because this particular endpoint uses the `jsonapi` format. Switching types from `string` to `time` is a problem because the API replies with a `string` but the package deserializer [expects a UNIX timestamp](https://github.com/google/jsonapi/blob/master/request.go#L22). It can only parse numbers as dates!

I guess I have to refactor `events_logs.go` to use `github.com/mitchellh/mapstructure` in order to be able to parse a `string` to `time`. That way we also only use one deserializer at the end. @trentrosenbaum what are your thoughts on this?

Reply:
So it would be good to focus on a single deserializer lib moving forward, but in this case I think we can use the expanded struct tag of `jsonapi:"attr,created_at,iso8601"` and this allows the use of a date format as provided by the fixture.  If I understand correctly the json returned is 
`created_at":"2019-05-11T17:53:06Z` and this is iso8601?

Reply:
@trentrosenbaum awesome that works!